maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/AbstractSurefireMojo.java

```diff
@@ -2289,16 +2321,13 @@ private void maybeAppendList( List<String> base, List<String> list )
     private void checkMethodFilterInIncludesExcludes( Iterable<String> patterns )
         throws MojoFailureException
     {
-        if ( patterns != null )
+        for ( String pattern : patterns )
         {
-            for ( String pattern : patterns )
+            if ( pattern != null && pattern.contains( "#" ) )
             {
-                if ( pattern != null && pattern.contains( "#" ) )
-                {
-                    throw new MojoFailureException( "Method filter prohibited in "
-                                                        + "includes|excludes|includesFile|excludesFile parameter: "
-                                                        + pattern );
-                }
+                throw new MojoFailureException( "Method filter prohibited in "
+                    + "includes|excludes|includesFile|excludesFile parameter: "
```

Comment:
`includesFile|excludesFile` should not be supported?

Reply:
good catch!
The commit 6e06d2aabb7d612dce1e966c456f2060eeff3eab did not employ any ITs in 2015.